fs/ggml/ggml.go

```diff
@@ -579,12 +579,16 @@ func (f GGML) GraphSize(context, batch uint64, kvCacheType string) (kv, partialO
 }
 
 func (llm GGML) VisionGraphSize() (weights, graphSize uint64) {
-	switch llm.KV().Architecture() {
-	case "mllama":
-		for _, layer := range llm.Tensors().GroupLayers()["v"] {
-			weights += layer.Size()
+	for name, layer := range llm.Tensors().GroupLayers() {
+		if strings.HasPrefix(name, "v.") {
+			for _, tensor := range layer {
+				weights += tensor.Size()
+			}
 		}
+	}
 
+	switch llm.KV().Architecture() {
```

Comment:
Consider using an if statement instead of a switch with a single case, as it would be more idiomatic Go for this scenario and potentially improve readability.